### PR TITLE
bump feat: evm chain support

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/37308523/CSI-1584

### CODE REVIEW

**Comment by irshadCypherock** [comment]
Task linked: [CU-85zrja2x6 Add support for Arbitrum & Optimism](https://app.clickup.com/t/85zrja2x6)

File: cysync/src/utils/networkFees.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ const getFees = async (coinId: string) => {
       );
     }
 
-    const fees = value.dividedBy(1000000000).integerValue().toNumber();
+    const fees = value.dividedBy(1000000000).toNumber();
```
Comment (irshadCypherock) [inline cysync/src/utils/networkFees.ts:38]: Need to round off this value

File: cysync/src/pages/mainApp/sidebar/wallet/send/generalComponents/CustomSlider.tsx [hunk 2]
```diff
@@ -146,7 +146,8 @@ const CustomSlider: React.FC<CustomSliderProps> = ({
     handleTransactionFeeChangeSlider(v);
   };
 
-  const stepValue = step ?? (mediumFee < 5 ? 0.1 : 0.2);
+  const stepValue =
+    step ?? (mediumFee < 5 ? (mediumFee < 0.1 ? 0.00001 : 0.1) : 0.2);
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/send/generalComponents/CustomSlider.tsx:150]: Can we have a logic which can define the step from medium fees?

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: cysync/src/utils/networkFees.ts (hunk 1 above)
Comment (ujjwal-cyph) [reply cysync/src/utils/networkFees.ts]: changed. now using precision of 3-digits

File: cysync/src/pages/mainApp/sidebar/wallet/send/generalComponents/CustomSlider.tsx (hunk 2 above)
Comment (ujjwal-cyph) [reply cysync/src/pages/mainApp/sidebar/wallet/send/generalComponents/CustomSlider.tsx]: added. now using 1% as step value for slider

**Review by irshadCypherock — APPROVED** [review]

**Comment by ujjwal-cyph** [comment]
replaced by https://github.com/Cypherock/cysync-desktop/pull/383

**Pull request closed without merging**